modules/users/client/controllers/settings/manage-social-accounts.client.controller.js

```diff
@@ -7,7 +7,9 @@ angular.module('users').controller('SocialAccountsController', ['$scope', '$http
     // Check if there are additional accounts
     $scope.hasConnectedAdditionalSocialAccounts = function (provider) {
       for (var i in $scope.user.additionalProvidersData) {
-        return true;
```

Comment:
I guess it's a way to check whether the object has at least one key. Probably not the cleanest way, but to satisfy `guard-for-in` I had to put `hasOwnProperty` check.